src/pages/home/ReportScreen.js

```diff
@@ -351,6 +351,7 @@ function ReportScreen({
                                 setSkeletonViewContainerHeight(newSkeletonViewContainerHeight);
                             }}
                         >
+                            {/* TODO */}
```

Comment:
Dev comment?

Reply:
good catch, committed the wrong file 😅 Thanks @fabioh8010 !